app/posts/manage-teacher-training-applications/2020-03-27-reasons-for-rejection.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Reasons for rejection
+description: Helping providers give useful feedback to candidate's who have been rejected
+date: 2020-03-27
+---
+
+{% from "figure/macro.njk" import appFigure with context %}
+
+## Hypothesis
+
+If a candidate’s application is rejected, they have a chance to apply again using [Apply 2](/apply-for-teacher-training/apply-again).
+
+Currently the provider is just given a [free text field to provide feedback](/as-launched-26-nov-2019#reject-application).
+
+This is problematic because:
+
+- providers may only give candidates a small amount of unstructured feedback
+- different users are likely to provide varying levels of feeedback to candidates
+- it puts the onus on providers to type everything out manually on a repeated basis
+- it puts the onus on providers to remember all the reasons they are being rejected
+
+All of this could result in candidates not knowing why they were rejected and as a result not being motivated to apply again successfully in the future. And it doesn't empower candidates to appeal such decisions if providers make a mistake.
```

Comment:
```suggestion
This means a candidate might not know why they were rejected and won’t know how to improve their chances if they apply again. It might also discourage them from applying again. It doesn't empower candidates to appeal decisions if providers make a mistake.
```